### PR TITLE
Prepare release 3.0.0

### PR DESCRIPTION
**Pull request by GomathiselviS**
- kubernetes.core version updated to 3.0.0
 - Dropped support for ansible-core < 2.14
 - Docs generated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,20 @@ The collection includes a variety of Ansible content to help automate the manage
 
 This collection has been tested against following Ansible versions: **>=2.14.0**.
 
+For collections that support Ansible 2.9, please ensure you update your `network_os` to use the
+fully qualified collection name (for example, `cisco.ios.ios`).
+Plugins and modules within a collection may be tested with only specific Ansible versions.
+A collection may contain metadata that identifies these versions.
```
Comment (alinabuzachis) [inline README.md:18]: I guess this can be probably removed.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,20 @@ OKD Collection Release Notes
 .. contents:: Topics
 
 
+v3.0.0
+======
```
Comment (alinabuzachis) [inline CHANGELOG.rst:9]: Can you please add a release_summary?

File: README.md (hunk 1 above)
Comment (GomathiselviS) [reply README.md:18]: Ensuring the use of fully qualified collection names aligns with one of the rules outlined in ansible-lint. I've noticed this requirement in a few other collections too. Could we rephrase this guideline? What are your thoughts?

Comment (alinabuzachis) [reply README.md:18]: Is there any collections that support Ansible 2.9?

Comment (GomathiselviS) [reply README.md:18]: I do not think so. I can rephrase the line while excluding the support for Ansible 2.9.

**Review by hakbailey — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**